packages/p2p/src/utils/server_time.ts

```diff
@@ -1,6 +1,18 @@
 import moment from 'moment';
 import { WS } from '@deriv/shared';
-import { PromiseClass } from './utility';
+
```

Comment:
We have another server-time util function in p2p. Can you please check if we can combine it? If not, we can rename this file.

Reply:
@farrah-deriv I did a quick check, It seems like we pass the value around between them 😅

From cashier to general store and from the general store to the utility in p2p but at the end they are all the same. I need to investigate it more to make sure 🤔 

Can we for now keep the legacy code as it was? I will pick this up after this PR 🙏🏻

Reply:
noted @farzin-deriv